### PR TITLE
[PWX-33177] Skip pool expansions when app has only FBDA volumes.

### PR DESCRIPTION
**Pull request by vinayakshnd**
Since FBDA volumes do not get created on PX pool, px pool expansion in case of FBDA volumes is not required.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

Successful job run: https://jenkins.pwx.dev.purestorage.com/job/Users/job/Vinayak/job/FBDA-MultiTenancy/29/console

### CODE REVIEW

File: tests/common.go [hunk 1]
```diff
@@ -1693,33 +1693,43 @@ func ValidatePureVolumeLargeNumOfClones(ctx *scheduler.Context, errChan ...*chan
...
-			err = fmt.Errorf("length of pools should be greater than 0")
-			processError(err, errChan...)
-		}
-		for _, pool := range pools {
-			initialPoolSize := pool.TotalSize / units.GiB
-			err = Inst().V.ResizeStoragePoolByPercentage(pool.Uuid, opsapi.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, 20)
+			pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
 			if err != nil {
-				err = fmt.Errorf("error initiating pool [%v ] %v: [%v]", pool.Uuid, opsapi.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, err.Error())
+				err = fmt.Errorf("error getting storage pools list. Err: %v", err)
```
Comment (vprabhakar-px) [inline tests/common.go:1708]: should we fail here instead of log.Error ?

File: tests/common.go [hunk 2]
```diff
@@ -1693,33 +1693,43 @@ func ValidatePureVolumeLargeNumOfClones(ctx *scheduler.Context, errChan ...*chan
...
 				if err != nil {
-					err = fmt.Errorf("pool [%v] %v failed. Error: %v", pool.Uuid, opsapi.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, err)
-					log.Error(err)
+					err = fmt.Errorf("error initiating pool [%v ] %v: [%v]", pool.Uuid, opsapi.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, err.Error())
+					log.Error(err.Error())
+				} else {
+					err = waitForPoolToBeResized(initialPoolSize, pool.Uuid)
+					if err != nil {
+						err = fmt.Errorf("pool [%v] %v failed. Error: %v", pool.Uuid, opsapi.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, err)
+						log.Error(err)
```
Comment (vprabhakar-px) [inline tests/common.go:1727]: can we fail instead of log.Error?

**Review by vprabhakar-px — APPROVED** [review]
LGTM

File: tests/common.go (hunk 1 above)
Comment (vinayakshnd) [reply tests/common.go:1708]: This is old piece of code not changed by this PR. This URL shows exact changes done as part of current PR: https://github.com/portworx/torpedo/pull/2456/files?diff=split&w=1

File: tests/common.go (hunk 2 above)
Comment (vinayakshnd) [reply tests/common.go:1727]: This is old piece of code not changed by this PR. This URL shows exact changes done as part of current PR: https://github.com/portworx/torpedo/pull/2456/files?diff=split&w=1

**Comment by vinayakshnd** [comment]
Thanks @vprabhakar-px

**Comment by javedlingasur-px** [comment]
Merging the code as per PTX-23855.

**Pull request merged**